style/js/shield_text.js

```diff
@@ -105,17 +105,16 @@ function layoutShieldText(text, padding, bounds, textLayoutFunc, maxFontSize) {
   var yBaseline;
 
   switch (textConstraint.valign) {
-    case VerticalAlignment.Middle:
-      yBaseline = padTop + (availHeight - textHeight) / 2;
-      break;
     case VerticalAlignment.Top:
       yBaseline = padTop;
       break;
     case VerticalAlignment.Bottom:
       yBaseline = padTop + availHeight - textHeight;
       break;
     default:
-      return null; //Code error, should never happen
+    case VerticalAlignment.Middle:
+      yBaseline = padTop + (availHeight - textHeight) / 2;
+      break;
```

Comment:
I guess I shouldn't be surprised that Javascript allows the `default:` case to be in the middle rather than last.  Unless there's some good reason I'm not grasping for the exact order as is, can we flip it around so the default case is last?
```
    case VerticalAlignment.Middle:
    default:
      yBaseline = padTop + (availHeight - textHeight) / 2;
      break;
```
Seems a bit clearer and the convention I'm used to seeing, even if JavaScript does allow just randomly chucking it in wherever 🤠.